deploy_to_openshift.sh

```diff
@@ -30,7 +30,7 @@ select_stress_profile() {
   heavy_containers_msg_per_sec=1000
   low_containers_msg_per_sec=10
   number_of_log_lines_between_reports=10;
-  maximum_logfile_size=10485760;
+  maximum_logfile_size=52428800;  # default file size
```

Comment:
@ajaygupta978 it looks like what you are trying to do is "not set" the log file rotation for your tests ... lete me suggest a better approach. I suggest revoking this change to maximum_logfile_size and instead adding a flag somewhere here https://github.com/ViaQ/cluster-logging-collector-benchmarks/blob/main/deploy_to_openshift.sh#L7 that allows controlling whether `maximum_logfile_size` is set by the benchmark or not. Actually, to make things even easier you can default to that flag to false, so it by default it will not change the `maximum_logfile_size` from default OCP value.

Reply:
@eranra Yes you are right. Thanks for suggestion now I think I have better suggestion. Instead of flag we can add `max_log_size` variable   as command line argument  which can be set by user based on selected profile. Currently to change value I need to do change in source code. By default it will take OCP value.

Reply:
@ajaygupta978 agree this can force the value ... I am just suggesting that for example if someone put a zero in that value we will not even call the function to set rate limits and use what ever comes out of the box from OCP.